src/utils/router_state.rs

```diff
@@ -404,17 +422,25 @@ pub fn get_possible_flights(
         error!("No route found");
         return Err("Route between vertiports not found".to_string());
     }
+    //1.1 Create a sorted vector of vertiports nearest to the departure vertiport (in case we need to create a deadhead flight)
```

Comment:
can this can be moved to just above the deadhead flight computation? don't want to compute this if we don't have to

Reply:
I had it there originally when I thought it's a good idea to go through 2 cycles of ` for i in 0..num_flight_options {`, but in the current implementation I went with only one cycle, so "Sorted vector of vertiports" needs to be computed before this cycle. 
One cycle is IMO better as it will try to fill all departure time options with some flight (normal or including deadhead).

E.g. If there are possible departure times 13:00, 13:05, 13:10 and let's say only 13:10 can have normal flight, but 13:00 can be available with deadhead flight.

In current implementation we would return both 13:00 (with deadhead) and 13:10 (normal) and customer could maybe choose if they want earlier but more expensive or later but cheaper.

With 2 cycles - we would return only 13:10 and wouldn't go through deadhead cycle at all. And in this case we can move "Sorted vector of vertiports" before the second( deadhead) cycle and not run it when we skip deadhead flights.

What is better?